### PR TITLE
Remove atmosphere as websocket's provider

### PR DESCRIPTION
**Pull request by pablojan**
Atmosphere has been use for last two years to provide a reliable transport layer for the client-server communications. However, at this moment it seems better to remove it because the library is not longer maintained and its integration with wave's code base is quite complex. Another reason is the lack of native clients of atmosphere for iOS and Android.

I think we can provide easily those features we liked in atmosphere:
- In real environments it is required to do keep alive pings through websocket to avoid network issues.
- Long-polling alternative is becoming less necessary these days as long as websocket is now widely adopted. Anyway, I think to implement long polling is not so hard.

### CODE REVIEW

File: wave/src/main/java/org/apache/wave/box/server/rpc/InitialsAvatarsServlet.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.awt.image.BufferedImage;
+import java.io.IOException;
+
+/**
+ * A servlet for fetching the users Initials Avatar.
+ */
+@Singleton
+public final class InitialsAvatarsServlet extends HttpServlet {
```
Comment (wisebaldone) [inline wave/src/main/java/org/apache/wave/box/server/rpc/InitialsAvatarsServlet.java]: fixed in other pr

File: wave/src/main/java/org/apache/wave/box/server/rpc/InitialsAvatarsServlet.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  private static final Log LOG = Log.get(InitialsAvatarsServlet.class);
+  private BufferedImage DEFAULT;
+
+  @Inject
+  public InitialsAvatarsServlet() throws IOException {
+    DEFAULT = ImageIO.read(Resources.getResource("org/apache/wave/box/server/rpc/InitialsAvatarDefault.jpg"));
```
Comment (wisebaldone) [inline wave/src/main/java/org/apache/wave/box/server/rpc/InitialsAvatarsServlet.java]: fixed in other pr

Comment (vega113) [inline wave/src/main/java/org/apache/wave/box/server/rpc/InitialsAvatarsServlet.java]: thanks

**Comment by wisebaldone** [comment]
Please pull from the branch you based this off to get those little edits corrected. Other than the style issue LGTM and thank you for this.

**Comment by pablojan** [comment]
Closing due to changes were made on top of another PR instead of master. Check new PR

**Comment by vega113** [comment]
I guess you could rebase your branch instead of opening a new one.

**Pull request closed without merging**